src/main/java/kg/jarkyn/server/Resource/ResourceReader.java

```diff
@@ -0,0 +1,23 @@
+package kg.jarkyn.server.Resource;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
+public class ResourceReader {
+
+    public static byte[] readFile(String filePath) {
+        try {
+            return Files.readAllBytes((new File(filePath)).toPath());
+        } catch (IOException e) {
+            e.printStackTrace();
+            return new byte[0];
+        }
+    }
+
+    public static List<String> readFolder(String folderPath) {
```

Comment:
The name `readFolder` may be a bit too general here. I, for one had to dive into the implementation to see what it does.